### ISSUE TITLE
[accessibillity] Basic accessibility for select component

### ISSUE THREAD

**Issue by jesca** [ElemeFE/element#5945]
Select component should be accessible by the keyboard. This includes:
1. Escape should close the dropdown and focus should be reset to the select element
2. Upon selecting by the keyboard (clicking enter), focus should be reset to the select element
3. A focused select element should have a clear visible focus outline (or change or border color).. something to differentiate it from being non-focused
4
